awx/main/models/notifications.py

```diff
@@ -228,8 +228,6 @@ def build_notification_running_message(self):
 
     def send_notification_templates(self, status_str):
         from awx.main.tasks import send_notifications  # avoid circular import
-        if status_str not in ['succeeded', 'failed', 'running']:
```

Comment:
I think you might've removed this on accident.

Reply:
Oops, sorry about that!  Fixed it.